### PR TITLE
Random showcase

### PR DESCRIPTION
**Pull request by seanyoung247**
# Hackathon Git Labs Project PR 🎉🎉

## PR contains:
- Doesn't add any features.
- I've refactored the JavaScript to use templates for both showcase and community sections.
- Refactored to download emojis once.
- Reordered the code a bit.

If this is too much reordering of working code feel free to reject.

## Hacktoberfest 2022:
- [x] Does this PR follow the `intermediate.md` guide and submitted as part of Hacktoberfest 2022?

## Breaking changes
- None

## Screenshots
- None

### CODE REVIEW

**Review by andrewdempsey2018 — APPROVED** [review]
Hi Sean, this is awesome, cheers! 🥇

**Comment by andrewdempsey2018** [comment]
@auxfuse There's a fair amount in this, I have reviewed thoroughly, tested and am happy to merge. I'll give it an hour before I do so in case you have any comments. Otherwise I'll go ahead.

File: index.html [hunk 1]
```diff
@@ -101,9 +148,6 @@ <h3>Meet the team:</h3>
             </ul>
         </div>
     </footer>
-
-    <!-- Custom JS Files -->
-    <script src="assets/js/main.js" defer></script>
```
Comment (auxfuse) [inline index.html:106]: The only thing I would argue here is the moving of the JS file script tag to the top of the HTML? Happy to be proven wrong here, however with the evolution of JS in recent years, maybe I'm just out of date? 😃 🦖

Comment (seanyoung247) [reply index.html:106]: The defer tag causes the script to be downloaded in parallel with the rest of the page (doesn't block) and wait until the from is ready for execution. So placing a defer script in the head should download and be ready as fast or faster than planning it at the end.

**Review by auxfuse — APPROVED** [review]
Huge PR 🙌 I learned a lot from reading through this, and genuinely very excited to see this merged 🤜

**Pull request merged**